### PR TITLE
add stats module

### PR DESCRIPTION
**Pull request by kmdalton**
Adding new console scripts for 
 - CChalf
 - CCanom
 - CCpred
 - Rsplit

This pull request adds a new stats submodule which houses the scripts. I added some basic tests for execution.

### CODE REVIEW

File: careless/stats/ccanom.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            "--method",
+            default="spearman",
+            choices=["spearman", "pearson"],
+            help=("Method for computing correlation coefficient (spearman or pearson)"),
+        )
+
+        self.add_argument(
+            "-b",
+            "--bins",
+            default=12,
```
Comment (JBGreisman) [inline careless/stats/ccanom.py:36]: It's a little funny to me that the default here is `12`, but the functions themselves have a default of `10`. This is fine to leave as is though -- up to you.

**Review by JBGreisman — APPROVED** [review]
This all looks good to me. In the future, we can refactor these implementations to avoid some of the repeated code, but that's a future improvement. The tests are pretty minimal and just confirm the code can run with a few commandline flags, but I think we can leave testing correctness to a future refactor as well. 

On the whole, looks fine to me.

File: careless/stats/ccanom.py (hunk 1 above)
Comment (kmdalton) [reply careless/stats/ccanom.py:36]: haha. it's a little silly, but i like 12 for no real reason.

**Comment by kmdalton** [comment]
Just added a `careless.completeness` script as well.

File: careless/stats/ccanom.py (hunk 1 above)
Comment (DHekstra) [reply careless/stats/ccanom.py:36]: Let's do 10 as a default, for my sanity. I've used 10 so far throughout.

Comment (DHekstra) [reply careless/stats/ccanom.py:36]: (I'm sure you can find an example where I did not use 10 but that's besides the point).

Comment (kmdalton) [reply careless/stats/ccanom.py]: okay i'll change it to 10.

Comment (JBGreisman) [reply careless/stats/ccanom.py]: sorry to be a stickler, but the help strings still say the default is 12

**Pull request merged**